packages/py-moose-lib/tests/test_olap_dictionary.py

```
@@ -0,0 +1,746 @@
+# approach mirrors exactly what `moose build` does in production.
+
+
+def _run_serializer(app_dir: str, moose_lib_root: str) -> dict:
+    """Run `python -m moose_lib.dmv2_serializer` and return the parsed infra map."""
+    env = {
+        **os.environ,
+        "PYTHONPATH": f"{moose_lib_root}{os.pathsep}{app_dir}",
+        "MOOSE_SOURCE_DIR": "app",
+        "IS_LOADING_INFRA_MAP": "true",
+    }
+    result = subprocess.run(
```

Ruff (0.15.9)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)